data/test/vtgate/unsupported_cases.txt

```diff
@@ -303,9 +303,13 @@
 "delete user from user join user_extra on user.id = user_extra.id where user.name = 'foo'"
 "unsupported: multi-table delete statement in sharded keyspace"
 
-# update changes index column
-"update music set id = 1 where id = 1"
-"unsupported: DML cannot change vindex column"
+# update changes primary vindex column
+"update user set id = 1 where id = 1"
+"unsupported: You can't update primary vindex columns. Invalid update on column: id"
+
+# update with complex set clause
+"update music set id = id + 1 where id = 1"
+"unsupported: Only values are supported. Invalid update on column: id"
```

Comment:
We need two more cases here right?
1. Update of a `Functional` non-primary vindex.
2. Update of a `Lookup` unowned vindex.

Reply:
Yup, I will add those.